internal/activity.go

```diff
@@ -45,6 +45,10 @@ type (
 		//
 		// WARNING: Task queue priority is currently experimental.
 		Priority Priority
+		// Retry policy for the activity. Note that the server may have set a different policy than the one provided
+		// when scheduling the activity. If the value is None, it means the server didn't send information about
```

Comment:
```suggestion
		// when scheduling the activity. If the value is nil, it means the server didn't send information about
```

Generally Go docs use `nil` not `None`